File: benchmarks/suite5/nested.rs
mod bench_runtime;
use bench_runtime::suite5_print_i64;

fn main() {
    let n: i64 = 450;
    let mut s: i64 = 0;
    let mut i: i64 = 0;
    while i < n {
        let mut j: i64 = 0;
        while j < n {
            s = s + (i * j + i) % 97;
            j += 1;
        }
        i += 1;
    }
    suite5_print_i64(s);
}

File: benchmarks/suite5/nested.go
package main


func main() {
	n := suite5OpaqueI64(450)
	var s int64
	var i int64
	for i = 0; i < n; i++ {
		var j int64
		for j = 0; j < n; j++ {
			s = s + (i*j+i)%97
		}
	}
	suite5PrintI64(s)
}
